book/templating.rst

```diff
@@ -1012,13 +1018,18 @@ but Symfony provides a more dynamic option via the ``asset`` Twig function:
 
         <img src="{{ asset('images/logo.png') }}" alt="Symfony!" />
 
-        <link href="{{ asset('css/blog.css') }}" rel="stylesheet" type="text/css" />
+        <link href="{{ asset('css/blog.css') }}"
+            rel="stylesheet"
+            type="text/css" />
 
     .. code-block:: html+php
 
-        <img src="<?php echo $view['assets']->getUrl('images/logo.png') ?>" alt="Symfony!" />
+        <img src="<?php echo $view['assets']->getUrl('images/logo.png') ?>"
+            alt="Symfony!" />
 
-        <link href="<?php echo $view['assets']->getUrl('css/blog.css') ?>" rel="stylesheet" type="text/css" />
+        <link href="<?php echo $view['assets']->getUrl('css/blog.css') ?>"
+            rel="stylesheet"
+            type="text/css" />
```

Comment:
Right now I'm on an external monitor, can't check, but looking at the character lines in my editor I think it would leave the scrollbar.